src/pages/sighting/EditSightingMetadata.jsx

```diff
@@ -141,12 +141,12 @@ export default function EditSightingMetadata({
           loading={loading}
           display="primary"
           onClick={async () => {
+            const properties = { ...defaultFieldValues };
```

Comment:
How is this different from const properties = defaultFieldValues?

(Unless perhaps defaultFieldValues is an array, in which case I take back the question.)

Reply:
this creates a shallow copy of the original object, which is helpful for preventing certain bugs as properties gets edited